packages/http-client-java/package.json

```diff
@@ -58,6 +58,7 @@
     "@autorest/codemodel": "~4.20.0",
     "@azure-tools/ts-genapi": "^1.0.7",
     "@microsoft/api-extractor-model": "^7.28.2",
+    "@typespec/http-client-java": "file:",
```

Comment:
this feels weird to have, what is the reason?

Reply:
hmm, this was auto-updated. Reverted the changes now.